### PR TITLE
Live blacklist updating

### PR DESCRIPTION
**Pull request by danfinlay**
I was able to merge the new Levenshtein work into this branch I started this morning that routinely polls Infura for an updated blacklist, and enforces it.

Was a little tricky because it requires inter-script communication, and apparently Chrome's extension auto-reloading does *not* reload a script's ability to inter-process communicate, so it wasn't until I fully disabled MetaMask and re-enabled it that this feature started working.

Definitely too much time trying to figure out why the code wasn't working.

Anyways, also refined @Zanibas's `isPhish` script to take an `opts` object instead of just a string, including an optional `updatedBlacklist` parameter, so that the function can receive the latest blacklist from Infura before it processes.

### CODE REVIEW

**Comment by flyswatter** [comment]
Probably of interest to @409H and @maurycy.  Once this is merged, we could deploy as soon as tomorrow with a blog post about the new blacklist API from Infura (serves 409H's list).

**Comment by Zanibas** [comment]
What will this mean for people updating then? Will the functionality work without the need for a restart?

**Comment by flyswatter** [comment]
Without restarting what? Chrome auto-updates without a browser restart, so people will get this update, and then when they load a page, a the `blacklister.js` content script will be initialized with each new page, it will ask the background page if it should blacklist, and if it replies yes, it will redirect.  The background script will simply begin polling Infura for a blacklist and caching it, and using it to revise how it informs new `blacklister.js` content scripts whether or not to redirect.

**Comment by tmashuang** [comment]
- [ ] In info.js, the Twitter link should be on a new line. Current CSS display is set to inline-block from fa, font-awesome, ```display: block``` should fix this for twitter link.
- [ ] Instance of Loading Spinner persisting after sending transactions. 
- [ ] Loading Spinner does not properly center
Note: When the extension URL is open in a window no popup appears when interaction with Dapps. Don't know if this is the intended functionality.

File: app/manifest.json [hunk 1]
```diff
@@ -55,8 +55,13 @@
     },
     {
       "run_at": "document_start",
-      "matches": ["http://*/*", "https://*/*"],
-      "js": ["scripts/blacklister.js"]
+      "matches": [
+        "file://*/*",
```
Comment (Zanibas) [inline app/manifest.json]: We're blacklisting files too?

Comment (flyswatter) [inline app/manifest.json]: I copied our other contentscript when I was trying to get it working. Removed this line since it doesn't really make sense.

**Review by 2-am-zzz — APPROVED** [review]
Don't really understand enough about the script communications to speak on that part, but everything else looks good!

**Comment by flyswatter** [comment]
If you ever can't reason about the code, the safest review strategy is to pull & try, but ty for the merge. It works!

**Pull request merged**